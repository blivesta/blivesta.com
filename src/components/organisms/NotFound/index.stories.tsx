import React from 'react';
// import styled from 'styled-components';
import { storiesOf } from '@storybook/react';
import NotFound from '.';

storiesOf(`organisms|NotFound`, module).add(`default`, () => <NotFound />);
