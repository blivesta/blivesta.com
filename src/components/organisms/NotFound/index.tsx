import * as React from 'react';

import Title from '../../atoms/Title';
import ContentBlock from '../../molecules/ContentBlock';

const NotFound = () => {
  return (
    <ContentBlock id="notFound" title="NOT FOUND">
      <Title>
        You just hit a route that doesn&#39;t exist...
        <br />
        the sadness.
      </Title>
    </ContentBlock>
  );
};

export default NotFound;
