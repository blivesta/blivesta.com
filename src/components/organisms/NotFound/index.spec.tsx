import * as React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import NotFound from '.';

describe('NotFound', () => {
  it('snapshot', () => {
    const tree = shallow(<NotFound />);
    expect(toJson(tree)).toMatchSnapshot();
  });
});
