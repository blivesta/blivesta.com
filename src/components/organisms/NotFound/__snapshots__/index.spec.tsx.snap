// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound snapshot 1`] = `
<ContentBlock
  id="notFound"
  title="NOT FOUND"
>
  <styled.h1>
    You just hit a route that doesn't exist...
    <br />
    the sadness.
  </styled.h1>
</ContentBlock>
`;
